Task_1/Parts/Class.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Task_1
+{
+    class Class
+    {
+        Pupil[] pupils = new Pupil[4];
+
+        //почему нельзя вставить new GoodPupil() в значения по умолчанию?
```

Comment:
In C# default parameter values must be defined on a compile step. In other words you are not allowed to use types that don't have a value in time on a compilation
https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/named-and-optional-arguments#optional-arguments